scan/lib/scan/test_command_generator.rb

```diff
@@ -31,8 +31,9 @@ def options
 
         options = []
         options += project_path_array
-        options << "-sdk '#{config[:sdk]}'" if config[:sdk]
-        options << destination # generated in `detect_values`
+        options << "-sdk '#{config[:sdk]}'" if config[:sdk] && !config[:build_for_testing]
+        options << destination unless config[:build_for_testing] # generated in `detect_values`
+        options << "  -sdk iphonesimulator ONLY_ACTIVE_ARCH=NO" if config[:build_for_testing]
```

Comment:
@samskiter isn't the idea of building without testing is to actually be able to run build product on different simulators?